src/main/java/com/aorri2/goodsforyou/product/domain/ProductStatus.java

```diff
@@ -0,0 +1,16 @@
+package com.aorri2.goodsforyou.product.domain;
+
+public enum ProductStatus {
+	BUY_AVAILABLE("구매 가능"),
```

Comment:
BUY Available 은 너무 판매자 입장에서 바라보는 상품의 상태 아닐까요???
이 경우엔 상품 그 자체를 나타내는 변수 이름이 더 어울려 보이는데, 종욱님은 어떻게 생각하세요?

Reply:
이 부분에 대해서 저도 다시 한번 생각해보니 
```suggestion
	TRADABLE("거래 가능"),
```
위와 같은 식으로 변경 가능할 것 같다 생각이 들었습니다. 혹시 멘토님이 생각하실때 더 나은 변수 이름이 있으시다면 알려주실수 있을까요?😊